routes/drones.js

```diff
@@ -1,34 +1,69 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
 
+const { create } = require("hbs");
+
+const Drone = require("../models/Drone.model");
+
 // require the Drone model here
 
-router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
+router.get("/drones", async (req, res, next) => {
+  try {
+    let drones = await Drone.find();
+    console.log(drones);
+    res.render("drones/list", { drones });
+  } catch (error) {
+    console.log(error);
+    next(error);
+  }
 });
 
-router.get('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
+router.get("/drones/create", (req, res) => res.render("drones/drones-create"));
```

Comment:
Here, the path to the file you want to render is incorrect. You have an `.hbs` file called `create-form` and that's the one you want your users to see.
`router.get("/drones/create", (req, res) => res.render("drones/create-form"));`